### PR TITLE
Simplification of the realtime profile, code de-duplication

### PR DESCRIPTION
**Pull request by yarda**
Signed-off-by: Jaroslav Škarvada jskarvad@redhat.com

### CODE REVIEW

File: profiles/realtime/variables.conf [hunk 1]
```diff
@@ -6,5 +6,6 @@
 #
 #
 
+isolated_cores = 1
```
Comment (jeremyeder) [inline profiles/realtime/variables.conf]: I think this should be commented out.

**Comment by yarda** [comment]
Removed in 960da9e177f8ce67933751a41f8ee52a53d24e2d

**Comment by yarda** [comment]
This pull request should be now up to date with the upstream tuned git and is ready for merge.

**Comment by jeremyeder** [comment]
I see the tuna "-i" option in the updated script.sh, which was missing from the initial PR.  Thanks to acme for catching that.

**Pull request merged**